### PR TITLE
Dom

### PR DESCRIPTION
**Pull request by DuaaH**
we did this :
.we change the name of variable from search to input in public/script.js .
.solve the conflect in  src/handler.js .
.adding query to get data by creating src/queries/getData.js .
.solve conflict in  src/router.js.
.connect router in  src/server.js  .

### CODE REVIEW

**Review by Muniralsharif — APPROVED** [review]

**Review by someyaaltous — APPROVED** [review]

**Review by SajaLahaleeh — APPROVED** [review]

File: public/script.js [hunk 1]
```diff
@@ -1,3 +1,36 @@
...
+};
+
+req("/search", data => {
+  if (input.value === "") {
+    alert("please enter the country");
+  }
+});
+
+//
+// search.addEventListener("click", event => {
```
Comment (jema28) [inline public/script.js]: Please delete these comments. Even if you're using them later, you shouldn't merge large comment blocks into master.

**Review by jema28 — CHANGES_REQUESTED** [review]
What is happening in this PR? Please add a description to the PR so I know what I'm testing :)

File: public/script.js (hunk 1 above)
Comment (jema28) [reply public/script.js]: There's a lot more than dom happening here and the PR name doesn't reflect the PR content.

File: src/queries/getData.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+const dbConnection = require('../db_connection');
+
+const getData = (country, cb) => {
```
Comment (jema28) [inline src/queries/getData.js:3]: Good getData query! There's a lot of repetition in this file though! You could refactor this make this getData function work for any country 😸

File: src/handler.js [hunk 3]
```diff
@@ -1,50 +1,63 @@
...
+  });
+};
 
 const notFound = (req, res) => {
   res.writeHead(404, {
-    "Content-Type": "text/html"
+    'Content-Type': 'text/html',
   });
-  res.end(`<h1>ERRORE 404 <br> Page Not Found</h1>`);
+  res.end('<h1>ERRORE 404 <br> Page Not Found</h1>');
```
Comment (jema28) [inline src/handler.js]: Spelling error here: 'ERROR'

File: src/queries/getData.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+const dbConnection = require('../db_connection');
```
Comment (jema28) [inline src/queries/getData.js]: This path is wrong, throws an error: `Cannot find module '../db_connection'`

**Review by jema28 — CHANGES_REQUESTED** [review]
The getData functionality isn't working, please check this :)

**Review by jema28 — APPROVED** [review]
You need to sort your eslint because it keeps changing from single to double to single quotes. Will merge for now though.

**Pull request merged**